templates/security-cover.html.twig

```diff
@@ -1,112 +1,19 @@
-                        </label>
-                        <div class="input-group input-group-flat">
-                            <input id="password" name="_password" type="password" tabindex="20" class="form-control" placeholder="{{ 'Password'|trans({}, 'TablerBundle') }}">
-                        </div>
-                    </div>
-                    {% block remember_me %}
-                        <div class="mb-3">
-                            <label class="form-check">
-                                <input id="remember_me" tabindex="30" name="_remember_me" type="checkbox" class="form-check-input">
-                                <span class="form-check-label">{{ 'Remember Me'|trans({}, 'TablerBundle') }}</span>
-                            </label>
-                        </div>
-                    {% endblock %}
-                    <div class="form-footer">
-                        <button type="submit" tabindex="40" class="btn btn-primary w-100">{{ 'Sign In'|trans({}, 'TablerBundle') }}</button>
-                    </div>
-                    {% block csrf_token %}
-                        <input type="hidden" name="_csrf_token" value="{{ csrf_token('authenticate') }}"/>
-                    {% endblock %}
-                    {% block login_form_end %}{% endblock %}
-                </form>
-            {% endblock %}
-            {% block login_social_auth %}
-                <div class="hr-text">{{ 'or'|trans({}, 'TablerBundle') }}</div>
-                <div class="card-body">
-                    <div class="row">
-                        <div class="col">
-                            <a href="#" class="btn btn-white w-100">
-                                <i class="icon fab fa-github text-github"></i> Github
-                            </a>
-                        </div>
-                        <div class="col">
-                            <a href="#" class="btn btn-white w-100">
-                                <i class="icon fab fa-twitter text-twitter"></i> Twitter
-                            </a>
-                        </div>
-                    </div>
-                </div>
-            {% endblock %}
-            
-            {% block login_actions %}
-                {% block registration %}
-                    {% if 'tabler_registration'|tabler_route != 'tabler_registration' %}
-                        <div class="text-center text-muted mt-3">
-                            <a href="{{ path('tabler_registration'|tabler_route) }}">
-                                {{ 'Register a new account'|trans({}, 'TablerBundle') }}
-                            </a>
-                        </div>
-                    {% endif %}
-                {% endblock %}
-            {% endblock %}
-        </div>
-    </div>
+
+    {% block container_classes %}container container-tight my-5 px-lg-5{% endblock %}
+    {% block card_classes %}{% endblock %}
+
+    {% block container_after %}
```

Comment:
that doesn't compile

Reply:
WDYM?

Reply:
You open a block and close a div: that feels / looks wrong 😁

Maybe it is the indentation that is actually the problem. 
Is `container_after` closed in line 20?

Reply:
Yes, `div` is opened with `block container_before` at L5
and closed at L12 with  `block container_after`.

The indents are like that to remind me that a DIV is open and not closed.
If the end of my file has one (or more) TABs, it means something is wrong.